packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Protocol-level requirements
+---
+
+### Code flow
+
+For OIDC applications destined for the OIN, you are building a "web" application with a dedicated server-side back end that is capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections.
+
+The exchange of the short-lived authorization code for a token requires a Client Secret. This is a value that only your Client and the authorization server know and must be protected from malicious parties.
+
+The server-side app also requires an end user because it relies on interaction with the end user's web browser to redirect the user and then receive the authorization code. The end user and the user agent do not see the access token issued by Okta or any part of the authentication response.
+
+The authorization code flow is the recommended flow for controlling access to an OIDC application in the OIN context.
+
+The authorization code flow looks like this for an OIN app:
+
+* Your end user (also known as the Resource Owner) opens their web browser (the User Agent) in order to access and work with your application (the Client), which in turn needs information held by various pieces of backend infrastructure (the Resource Server), usually an API or service. By clicking on a sign-in link or an icon on their Okta dashboard, the end user grants authorization to your application and initiates the authorization code flow
+* Your Client prepares an authentication request to authenticate the end user (or determine if the end user is already authenticated). This request is composed of specific required parameters and is sent to the Okta `/authorize` endpoint:
+  * `client_id` &mdash; this value is the Okta client ID created for you when you made your Okta app integration and is available in the app integration settings
+  * `nonce` &mdash; A value that is returned in the ID token to mitigate against replay attacks
+  * `redirect_uri` &mdash; this is the location where the response will be sent. This value, which must start with `https://`, is set in your Okta app integrations settings
+  * `response_type=code` &mdash; this informs Okta (as the authorization server) that you want to get back an access token and an ID token in exchange for the authorization code
```

Comment:
```suggestion
  * `response_type=code` &mdash; this informs Okta (as the authorization server) that you want to get back an  authorization code
```

Reply:
@kasturikumthekar-okta please confirm

Reply:
This is taken from https://openid.net/specs/openid-connect-basic-1_0.html#RequestParameters - unless I have interpreted this incorrectly.

Reply:
my take - this informs Okta (as the authorization server) that you want to get back an  authorization code that you will exchange to get back an access token and/or id token.

Reply:
@kasturikumthekar-okta confirmed original description is correct